src/status_im/ui/screens/chat/components/style.cljs

```diff
@@ -128,3 +145,53 @@
    :background-color (colors/get-color :ui-background)
    :border-top-width 1
    :border-top-color (colors/get-color :ui-01)})
+
+(defn new-input-bottom-sheet [window-height]
+  (merge
+    {:border-top-left-radius  20
+     :border-top-right-radius 20
+     :position                :absolute
+     :left                    0
+     :right                   0
+     :bottom                  (- window-height)
+     :height                  window-height
+     :flex                    1
+     :background-color        (quo2.colors/theme-colors quo2.colors/white quo2.colors/neutral-90)
+     :z-index                 1000}
+    (if platform/ios?
+      {:shadow-radius  16
+       :shadow-opacity 1
+       :shadow-color   "rgba(9, 16, 28, 0.04)"
+       :shadow-offset  {:width 0 :height -2}}
+      {:elevation 2})))
+
+(defn new-bottom-sheet-handle []
+  {:width            32
+   :height           4
+   :background-color (quo2.colors/theme-colors quo2.colors/black quo2.colors/white)
+   :opacity          0.05
+   :border-radius    100
+   :align-self       :center
+   :margin-top       8})
+
+(defn new-bottom-sheet-controls [insets]
+  {:flex-direction     :row
+   :padding-horizontal 20
+   :elevation          2
+   :z-index            2000
```

Comment:
why ?

Reply:
> why ?

Oh never mind the first comment, only the link was important.

It was just a suggestion that we should find a pattern for the z-index instead of using a big arbitrary value.
Otherwise, if other developers don't know about this, they might try every value from 0-2000, thinking why there view is still not coming on top even with a big `z-index`

Reply:
its only inside the parent view, so it doesn't make sense, we could use 1 and 2